docs/schemas.md

```diff
@@ -302,3 +302,18 @@ The following template provides an example with two CTA blocks and a terminal Co
   <!-- INSERT ANY VALID HTML HERE -->
 </div>
 ```
+
+### Desmos Interactive block
+
+#### Description
+
+The Desmos interactive block displays a Desmos graphing calculator. The user can add expressions if the `data-expressions` attribute is set to `true`. The schema user can define where the graph viewport is using `data-bottom`, `data-top`, `data-left`, and `data-right` attributes. The schema user can define the size of the Desmos interactive by passing an integer value into `data-width` and `data-height` attributes. The default is 400 pixles width and height. 
+
+#### Schema definition
+
+The following template provides an example with a Desmos interactive graphing calculator block. 
+
+```html
+<div class="os-raise-ib-desmos" data-wait-for-event="eventnameY"  data-height="500" data-width="600"  data-equations='["(1,2)","x=5"]' data-expressions="true" data-top="50" data-bottom="-50" data-left="-50" data-right="50">
```

Comment:
One more comment on schema: Even though we don't parse for it, let's include the `data-schema-version="1.0"` similar to the other blocks so WW includes that when they copy/paste the snippet.